File: 7.py
import math

def f(x):
    return x**5 / math.sqrt(4 - x**2)

def trapezoidal_integration(a, b, n):
    h = (b - a) / n
    total = 0.5 * (f(a) + f(b))

    for i in range(1, n):
        x = a + i * h
        total += f(x)

    return total * h

a = 0.0
b = 2.0
n = 10000
b = 1.999999  

result = trapezoidal_integration(a, b, n)

print(f"Результат интегрирования методом трапеций: {result:.10f}")

File: 7.cpp
#include <iostream>
#include <cmath>
#include <iomanip>

using namespace std;

double f(double x) {
    return pow(x, 5) / sqrt(4 - x*x);
}

double trapezoidal_integration(double a, double b, int n) {
    double h = (b - a) / n;
    double sum = 0.5 * (f(a) + f(b));
    
    for (int i = 1; i < n; ++i) {
        double x = a + i * h;
        sum += f(x);
    }
    
    return sum * h;
}

int main() {
    double a = 0.0;    
    double b = 2.0;    
    int n = 10000;     
    b = 1.999999;
    
    double result = trapezoidal_integration(a, b, n);
    
    cout << "Результат интегрирования методом трапеций: " 
         << setprecision(10) << result << endl;
    
    
    
    return 0;
}
